ci/tests/e2e-ansible.sh

```diff
@@ -158,9 +158,10 @@ operator-sdk new memcached-operator --api-version=ansible.example.com/v1alpha1 -
 pushd memcached-operator
 # Add a second Kind to test watching multiple GVKs
 operator-sdk add crd --kind=Foo --api-version=ansible.example.com/v1alpha1
-sed -i 's|{{ pull_policy.default..Always.. }}|Always|g' deploy/operator.yaml
+
```

Comment:
Nit: is there a reason for all of the introduced newline spacing between the `sed` commands? The new lines make it more difficult to review the changes since the removals and additions are now on separate lines.